src/components/admin/article/ArticleController.jsx

```diff
@@ -115,7 +115,7 @@ export default class ArticleController extends FalcorController {
       const image = article.image || '';
       const authors = _.map(article.authors, author => author);
 
-      this.safeSetState({
+      this.safeSetoState({
```

Comment:
Interesting... Don't know how this happened haha, but definitely a bug ;). Hopefully CI should catch this in build phase too?

Reply:
oof good catch.

Reply:
How did CI not catch this though... Worrying lol, I guess E2E tests never reach there, very very scary lol